src/aatoolbox/datasources/codab.py

```diff
@@ -0,0 +1,88 @@
+"""Retrieve COD administrative boundaries."""
```

Comment:
Very general comment, but do we want to have a folder per datasource if num files>=2 and else just have a single `.py` file, or always encapsulate it by a dir?

Reply:
That's kind of what I was thinking, because I feel like it's overkill to have packages with just a single module. At the same time, maybe it's inconsistent to have some datasource modules be just within the datasource package while others have their own package. Maybe I'm starting to lean towards directories for everything. What about you?

Reply:
I would say directories. Feels like if we have both directories and single files, it becomes less organized and harder to understand

Reply:
I would also vote directories here. This was something I was just thinking about where we probably want to have a datasource utils directory (not sure if we want to explicitly call it that in case it confuses with the main utils directory). But things like the Area.py used in CDS is going to be useful across multiple APIs, and we would probably want most of that functionality if not all in a utils directory accessible to others.

Reply:
It depends which packages will use `Area`. If it's just CDS then it should stay there, but otherwise it could maybe go to `utils`? At any rate agreed, I will move COD AB to its own directory!

Reply:
Also that way the `__init__` file of each data source is a good place to explain what exactly the datasource is

Reply:
Great. Specific to `Area`, I also wanted to use it for another datasource so moved it to `utils` in #21